core/domain/platform_parameter_domain.py

```diff
@@ -791,20 +765,6 @@ def validate(self) -> None:
             raise utils.ValidationError(
                 'Unsupported data type \'%s\'.' % self._data_type)
 
```

Comment:
Do we need to validate that the name is one of the ones in the given list, or is that not necessary? (I.e. if some code creates a platform parameter domain object with a name that's 'abcdef', should that be caught here? Or is it impossible for that to happen?)

Reply:
That would not be required, according to the updated code we can only create platform-parameters of type platform_parameter_list.PlatformParams.
Thanks